docs/docs/basics/routes.md

```diff
@@ -362,6 +362,16 @@ Response onRequest(RequestContext context, String id) {
 }
 ```
 
+Dart Frog also supports nested dynamic routes. For example, if you create a file called, `routes/users/[userId]/posts/[postId].dart`, then it will be accessible at `/users/alice/posts/1`, `/users/sam/posts/42`, etc.
```

Comment:
```suggestion
Dart Frog also supports nested dynamic routes. For example, if you create a file called, `routes/users/[userId]/posts/[postId].dart`, then it will be accessible at `/users/alice/posts/1`, `/users/sam/posts/42`.
```